WebSocketTransportSettingsTest does not compile in TOF

Recent changes in PR #546 introduced code that does not compile in ToF (probably because the Assert types in ToF do not support Func).  When testing the property getter, it should probably assign the get to a local variable rather than return it (i.e. be an Action not a Func).

"E:\ProjectK\src\QA\ToF\tests\FX\Conformance\System.Private.ServiceModel\4.0.0.0\WcfOpen\unit\System.ServiceModel.Http\System.ServiceModel.Http.Tests.csproj" (default target) (1) ->
(CoreCompile target) -> 
  e:\ProjectK\src\NDP\FxCore\WcfOpen\src\System.ServiceModel.Http\tests\Channels\WebSocketTransportSettingsTest.cs(21,60): error CS0201: Only assignment, call, increment, decrement, await, and new object expressions can be used as a statement [E:\ProjectK\src\QA\ToF\tests\FX\Conformance\System.Private.ServiceModel\4.0.0.0\WcfOpen\unit\System.ServiceModel.Http\System.ServiceModel.Http.Tests.csproj]

    1 Warning(s)
    1 Error(s)

--- a/src/System.ServiceModel.Http/tests/Channels/WebSocketTransportSettingsTest.cs
+++ b/src/System.ServiceModel.Http/tests/Channels/WebSocketTransportSettingsTest.cs
@@ -18,6 +18,8 @@ public static class WebSocketTransportSettingsTest
     public static void DisablePayloadMasking_Property_Get_PNSE_Throws()
     {
         var setting = new WebSocketTransportSettings();
-        Assert.Throws<PlatformNotSupportedException>(() => setting.DisablePayloadMasking);
+        bool disablePayloadMasking = false;
+        Assert.Throws<PlatformNotSupportedException>(() => disablePayloadMasking = setting.DisablePayloadMasking);
+        Assert.False(disablePayloadMasking);
     }
 }

